packages/calypso-e2e/src/lib/pages/published/published-post-page.ts

```diff
@@ -83,10 +85,13 @@ export class PublishedPostPage {
 	 *
 	 * @param {string} title Title text to check.
 	 */
-	async validateTitle( title: string ): Promise< void > {
-		const dash = /-/g;
-		title = title.replace( dash, '–' );
-		await this.page.waitForSelector( `:text("${ title }")` );
+	async validateTitle( title: string ) {
+		// The dash is used in the title of the published post is
+		// not a "standard" dash, instead being U+2013.
+		// We have to replace any expectatiosn of "normal" dashes
+		// with the U+2013 version, otherwise the match will fail.
+		const sanitizedTitle = title.replace( /-/g, '–' );
```

Comment:
Unnecessary change, but figured I'd add an explanation and simplify down to 2 lines of code.